README.md

```diff
@@ -95,9 +96,11 @@ Astro defines a controller inside of a requirejs module. The module exposes a pr
     * `app/scaffold-controllers/welcomeController.js` manages the navigation and layout of the welcome screen
```

Comment:
maybe fix the path for this and the bottom two - it's really relevant to this PR but would save another tiny one? :)